web/react/components/activity_log_modal.jsx

```diff
@@ -49,16 +50,23 @@ export default class ActivityLogModal extends React.Component {
     }
     onShow() {
         AsyncClient.getSessions();
+
+        $(ReactDOM.findDOMNode(this.refs.modalBody)).css('max-height', $(window).height() - 300);
+        if ($(window).width() > 768) {
+            $(ReactDOM.findDOMNode(this.refs.modalBody)).perfectScrollbar();
+        }
```

Comment:
@asaadmahmoodspin Are you okay with just making the max-height for modals at 80% or something? I agree with Joram that we want to move away from the jquery stuff and I don't think having the height set to the window height - 300 gets us anything special that can't be achieved by just using a percent height.